### PR TITLE
chore: get ai package ready for release

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-ai/src/components/AIConversation/AIConversationProvider.tsx [hunk 1]
```diff
@@ -1,27 +1,25 @@
 import React from 'react';
 
-import { ElementsProvider } from '@aws-amplify/ui-react-core/elements';
-
 import { AIConversationInput, AIConversationProps } from './types';
 import { defaultAIConversationDisplayTextEn } from './displayText';
 import {
-  ConversationDisplayTextProvider,
```
Comment (dbanksdesign) [inline packages/react-ai/src/components/AIConversation/AIConversationProvider.tsx:8]: This is just alphabetizing the imports

File: packages/react-ai/src/components/AIConversation/AIConversationProvider.tsx [hunk 2]
```diff
@@ -53,51 +50,47 @@ export const AIConversationProvider = ({
     ...displayText,
   };
   return (
-    <ElementsProvider elements={elements}>
```
Comment (dbanksdesign) [inline packages/react-ai/src/components/AIConversation/AIConversationProvider.tsx:56]: This just removes the Elements provider

File: packages/react-ai/src/components/AIConversation/context/ControlsContext.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ export interface ControlsContextProps {
     {
       handleSubmit: (e: React.FormEvent<HTMLFormElement>) => void;
       allowAttachments?: boolean;
+      isLoading?: boolean;
```
Comment (dbanksdesign) [inline packages/react-ai/src/components/AIConversation/context/ControlsContext.tsx:11]: Passing this in so customers can disable the input if the conversation is loading

File: packages/react/src/components/ThemeProvider/ComponentStyle.tsx [hunk 4]
```diff
@@ -14,10 +14,6 @@ interface ComponentStyleProps extends React.ComponentProps<'style'> {
   componentThemes: BaseComponentTheme[];
 }
 
-/**
- * @experimental
```
Comment (dbanksdesign) [inline packages/react/src/components/ThemeProvider/ComponentStyle.tsx:18]: going GA for these things too. why not

**Review by thaddmt — APPROVED** [review]
WOO! 🙌🎊

**Review by calebpollman — APPROVED** [review]

**Pull request merged**